### PR TITLE
CLOUDP-120225: [AtlasCLI] we should not show credential error when running "atlas login"

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-120225

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/root/atlas/builder.go [hunk 1]
```diff
@@ -218,6 +218,7 @@ func shouldCheckCredentials(cmd *cobra.Command) bool {
 		fmt.Sprintf("%s %s", atlas, "completion"), // completion commands do not require credentials
 		fmt.Sprintf("%s %s", atlas, "config"),     // user wants to set credentials
 		fmt.Sprintf("%s %s", atlas, "auth"),       // user wants to set credentials
+		fmt.Sprintf("%s %s", atlas, "login"),      // user wants to set credentials
```
Comment (gssbzn) [inline internal/cli/root/atlas/builder.go:221]: thanks for the fix can you check if mongocli has the same problem please?

Comment (gssbzn) [reply internal/cli/root/atlas/builder.go:221]: also I think you should be able to do this in the search by name since `"login"` is a terminal command

Comment (andreaangiolillo) [reply internal/cli/root/atlas/builder.go:221]: no, it doesn’t. We are not validating the credentials at the build level but only in the atlas command

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**